01_Jupyter_notebook_intro/index.md

```diff
@@ -85,6 +85,12 @@ Po úspěšné instalaci lze Jupyter Notebook spustit jednoduchým příkazem:
 $ jupyter-notebook
 ```
 
+Pokud tento příkaz čirou náhodou nefunguje, vyzkoušej následující:
```

Comment:
To „čirou náhodou“ mi přijde možná trochu popichující. Vypadá to, že to je běžný stav. Při instalaci Homebrewem funguje ten `jupyter-notebook`, při instalaci PIPem funguje `python3 -m notebook`. Věřím, že na Windows to může být podobné.